admin-dev/themes/default/template/helpers/list/list_header.tpl

```diff
@@ -352,6 +352,37 @@
 											});
 										</script>
 									</div>
+								{elseif $params.type == 'datetime'}
+									<div class="date_range row">
+										<div class="input-group fixed-width-md center">
+											<input type="text" class="filter datetimepicker date-input form-control" id="local_{$params.id_date}_0" name="local_{$params.name_date}[0]"  placeholder="{l s='From'}" />
+											<input type="hidden" id="{$params.id_date}_0" name="{$params.name_date}[0]" value="{if isset($params.value.0)}{$params.value.0}{/if}">
+											<span class="input-group-addon">
+												<i class="icon-calendar"></i>
+											</span>
+										</div>
+										<div class="input-group fixed-width-md center">
+											<input type="text" class="filter datetimepicker date-input form-control" id="local_{$params.id_date}_1" name="local_{$params.name_date}[1]"  placeholder="{l s='To'}" />
+											<input type="hidden" id="{$params.id_date}_1" name="{$params.name_date}[1]" value="{if isset($params.value.1)}{$params.value.1}{/if}">
+											<span class="input-group-addon">
+												<i class="icon-calendar"></i>
+											</span>
+										</div>
+										<script>
+                                            $(function() {
```

Comment:
The bad indentation is here.
More, I don't know if this datetime helper needs to be added on the `new-theme` theme?
PIng @matks

Reply:
As far as I know, HelperList does not work on `new-theme` (so => no need to) 🤔 but it would be better to check